tools/getmoduleversion.py

```diff
@@ -0,0 +1,22 @@
+import os,re
+
+def get_version():
+  node_version_h = os.path.join(
+    os.path.dirname(__file__),
+    '..',
+    'src',
+    'node_version.h')
+
+  f = open(node_version_h)
+
+  regex = '^#define NODE_MODULE_VERSION [0-9]+'
+
+  for line in f:
+    if re.match(regex, line):
+      major = line.split()[2]
+      return major
+  
+  raise Exception('Could not find pattern matching %s' % regex)
+
+if __name__ == '__main__':
+  print get_version()
```

Comment:
If you add `from __future__ import print_function` at the top, then you can write `print(get_version())` here.  A little more future-proof.